src/settings.ts

```diff
@@ -17,7 +22,8 @@ export enum METRIC_NAMES {
   PIN_SUCCEEDED = 'pin_succeeded',
   PIN_FAILED = 'pin_failed',
 
-  RETRY_PROCESSING = 'retry_processing',
+  READY_PROCESSING = 'ready_processing', // is this a good name ??
```

Comment:
also don't forget to remove these comments

Reply:
yep removed!